### DISCUSSION TITLE
Sharing state between two Select components

### DISCUSSION THREAD

**Discussion by binhxn** [ariakit/ariakit#1889] [Questions] [answered]
Hello 👋 

I was wondering if it's possible to share state between two Select components? I have a scenario where I want to display a Select on a large viewport, but as I scale it down to mobile viewport, I want the Select to be displayed in a Dialog/Modal. However, I'm running into a few issues when sharing the same state between the two Selects:
1. I'm unable to open the first SelectPopover. 
2. When the Dialog is open and I open the second SelectPopover, hovering over the popover closes it before I can select a value.

My expectation would be that both Selects would be able to reference the same selected value. I've created a [Codesandbox](https://codesandbox.io/s/sweet-shtern-rd1dr9?file=/src/App.tsx&resolutionWidth=873&resolutionHeight=927) as reference. Here's the video of the issues mentioned above:

https://user-images.githubusercontent.com/8155387/191885283-851936a7-5507-4eb8-ba06-498ab4313103.mp4

Would love to hear your feedback on this!

**Comment by diegohaz** [comment, marked as answer]
`useSelectState` doesn't support multiple instances. But you can share state between multiple `useSelectState`.

In your case, I think it makes sense sharing only the `value` state:
```jsx
const [value, setValue] = useState("Apple");
const select1 = useSelectState({ value, setValue });
const select2 = useSelectState({ value, setValue });
```
